tailwind.config.ts

```diff
@@ -1,14 +1,43 @@
 import type { Config } from "tailwindcss";
+import { fontFamily } from "tailwindcss/defaultTheme";
 
-const config: Config = {
+const config = {
```

Comment:
this change is not necessary, we can revert it
```suggestion
const config: Config = {
```